scripts/nginx/falae.conf.erb

```diff
@@ -26,17 +26,17 @@ server {
   server_tokens off;
 
   # certs sent to the client in SERVER HELLO are concatenated in ssl_certificate
-  ssl_certificate /etc/letsencrypt/live/falaeapp.org/fullchain.pem; # managed by Certbot
-  ssl_certificate_key /etc/letsencrypt/live/falaeapp.org/privkey.pem; # managed by Certbot
-  ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot
+  ssl_certificate <%= ssl_certificate_path %>;
+  ssl_certificate_key <%= ssl_certificate_key_path %>;
+  ssl_dhparam <%= ssl_dhparam_path %>;
   ssl_session_timeout 1d;
   ssl_session_cache shared:SSL:50m;
   ssl_session_tickets off;
 
 
   # modern configuration. tweak to your needs.
   ssl_protocols TLSv1.2;
-  ssl_ciphers 'ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256';
+  ssl_ciphers '<%= ssl_ciphers %>';
```

Comment:
PS: I don't see this as sensitive information. No need to move it to ` .yaml` config file.